libs/components/src/lib/slider/slider.spec.ts

```diff
@@ -139,6 +140,31 @@ describe('vwc-slider', () => {
 		});
 	});
 
+	describe('connotation', function () {
+		const possibleConnotations = [Connotation.Accent,
+			Connotation.CTA
+		];
+
+		it('should leave connotation class empty if not set', async function () {
+			possibleConnotations.forEach(connotation => {
+				expect(getControlElement(element)
+					?.classList
+					.contains(connotation))
+					.toEqual(false);
+			});
+		});
+
+		it('should set a connotation class', async function () {
+			const connotation = possibleConnotations[1];
+			(element.connotation as Connotation) = connotation;
+			await elementUpdated(element);
+			expect(getControlElement(element)
+				?.classList
+				.contains(`connotation-${connotation}`))
+				.toEqual(true);
+		});
+	});
```

Comment:
I think we can write it like this for all connotations
```suggestion
		it.each(possibleConnotations)('should set the connotation class for "%s"', async function (connotation) {
			element.connotation = connotation;
			await elementUpdated(element);
			expect(getControlElement(element)
				?.classList
				.contains(`connotation-${connotation}`))
				.toEqual(true);
		});
	});
```